subgraph.rinkeby.yaml

```diff
@@ -146,4 +146,29 @@ dataSources:
         - event: TransferWithReferenceAndFee(address,uint256,indexed bytes,uint256,address)
           handler: handleTransferWithReferenceAndFee
       file: ./src/ethConversionProxy.ts
-  
\ No newline at end of file
+  - kind: ethereum/contract
+    name: ERC20EscrowToPay
+    network: rinkeby
+    source:
+      address: "0x8230e703B1c4467A4543422b2cC3284133B9AB5e"
+      abi: ERC20EscrowToPay
+      startBlock: 9669613
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      entities:
+        - Payment
+      abis:
+        - name: ERC20EscrowToPay
+          file: ./abis/ERC20EscrowToPay.json
+      eventHandlers:
+        - event: InitiatedEmergencyClaim(indexed bytes)
+          handler: handleInitiatedEmergencyClaim
+        - event: RequestFrozen(indexed bytes)
+          handler: handleRequestFrozen
+        - event: RevertedEmergencyClaim(indexed bytes)
+          handler: handleRevertedEmergencyClaim
+        - event: TransferWithReferenceAndFee(address,address,uint256,indexed bytes,uint256,address)
```

Comment:
Will it also work if the `TransferWithReferenceAndFee` event is no more sent from this contract as delegated but from the ERC20FeeProxy contract?

Reply:
Won't it duplicate the detected payments as we kept the ERC20FeeProxy handlers? Or is it just a transition phase?

Reply:
> Will it also work if the `TransferWithReferenceAndFee` event is no more sent from this contract as delegated but from the ERC20FeeProxy contract?

I understood it as it would catch the events from the ERC20FeeProxy regardless of whether it was delegated or not.   🤔

Reply:
When it is delegated, the event comes from the Escrow, in the payEscrow function.
When it is not delegated, the event comes from ERC20FeeProxy, in the _withdraw function.

The withdrawal payment events will only appear in the ERC20FeeProxy part of the subgraph, if that's the case, I need to check there if Escrow entity exists and update it.